config-linux.md

```diff
@@ -359,7 +359,12 @@ For more information, see the kernel cgroups documentation about [cpusets][cgrou
 The following parameters can be specified to set up the controller:
 
 * **`shares`** *(uint64, OPTIONAL)* - specifies a relative share of CPU time available to the tasks in a cgroup
-* **`quota`** *(int64, OPTIONAL)* - specifies the total amount of time in microseconds for which all tasks in a cgroup can run during one period (as defined by **`period`** below)
+* **`quota`** *(int64, OPTIONAL)* - specifies the replenished amount of time in microseconds for which all tasks in a cgroup can run during one period (as defined by **`period`** below)
+    A value of `-1` indicates that the cgroup does not have any bandwidth restriction in place (i.e., an unconstrained bandwidth group).
```

Comment:
I am not sure about having `-1` here.  On cgroup v2 `quota` and `period` are written together.  Does `-1` mean it has the same value as `period`?

Reply:
For cgroup v2, I suppose `-1` means `max`, which can be written together w/ `period`.

According to the the `cpu.max` section of [cgroup v2 doc](https://github.com/torvalds/linux/blob/master/Documentation/admin-guide/cgroup-v2.rst#cpu):
```
cpu.max
A read-write two value file which exists on non-root cgroups. The default is "max 100000".

The maximum bandwidth limit. It's in the following format:
$MAX $PERIOD
which indicates that the group may consume upto $MAX in each $PERIOD duration. "max" for $MAX indicates no limit. If only one number is written, $MAX is updated.
````

Reply:
I think this applies to all negative integers actually (i.e., not only `-1`). Any suggestions? Thanks!

Reply:
I would guess the question here is really why it's important to `burst` support that we include support for `-1` here too -- I'd say this is a breaking change, since the spec doesn't currently say anything about negative numbers, and instead implies that this optional field be left unspecified for no restriction.  Does `cgroupv1` support negative values?

To be clear, I agree that we should clarify this, especially that unspecified needs to be `max` in `cgroupv2`, but I'm not sure how it relates to `burst` except that it's adjacent.